### PR TITLE
fix: Creating team and preview URL fixes [CAL-1997] [CAL-1998]

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Team list preview URL was not taking into account org teams URLs and team creation wizard was showing wrong URL prefix on team slug field. Also, corrected Shell bottom "Settings" button to go to orgs profile.

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Mandatory Tasks
- [ ] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

**Review by sean-brydon — DISMISSED** [review]
Code looks good and seems to be working. we have some linting errors however.

File: packages/features/ee/organizations/lib/orgDomains.ts [hunk 1]
```diff
@@ -43,3 +43,7 @@ export function subdomainSuffix() {
   const urlSplit = WEBAPP_URL.replace("https://", "")?.replace("http://", "").split(".");
   return urlSplit.length === 3 ? urlSplit.slice(1).join(".") : urlSplit.join(".");
```
Comment (hariombalhara) [inline packages/features/ee/organizations/lib/orgDomains.ts:44]: Though it's existing code, I think we shouldn't give special treatment to 3 dots. We have already add an issue where someone was hosting with more than 3 dot domains.
https://github.com/calcom/cal.com/pull/9803

File: apps/web/pages/event-types/index.tsx [hunk 2]
```diff
@@ -739,9 +736,9 @@ const EventTypeListHeading = ({
         )}
         {profile?.slug && (
           <Link href={`${CAL_URL}/${profile.slug}`} className="text-subtle block text-xs">
-            {orgSlug
-              ? `${orgSlug}.${subdomainSuffix()}/${profile.slug}`
-              : `${CAL_URL?.replace("https://", "")}/${profile.slug}`}
+            {orgBranding
+              ? `${orgBranding.fullDomain.replace("https://", "").replace("http://", "")}${profile.slug}`
```
Comment (hariombalhara) [inline apps/web/pages/event-types/index.tsx:740]: nit: we should have a `orgBranding.domainOnly` or something that takes care of it already.

File: packages/features/ee/organizations/pages/settings/profile.tsx [hunk 3]
```diff
@@ -46,14 +43,19 @@ const OrgProfileView = () => {
...
 
   useLayoutEffect(() => {
     document.body.focus();
   }, []);
 
+  const form = useForm({
+    resolver: zodResolver(orgProfileFormSchema),
+  });
+
+  if (!orgBranding) return null;
```
Comment (hariombalhara) [inline packages/features/ee/organizations/pages/settings/profile.tsx:57]: Conditional before a hook will cause error.

**Review by hariombalhara — CHANGES_REQUESTED** [review]
Left some comments.

**Review by hariombalhara — APPROVED** [review]
Fixed the blocker issue. LGTM !!

**Pull request merged**